docs/Upload.md

```diff
@@ -0,0 +1,51 @@
+### `uploadFactory(options: Options): function(file, imageId?)`
+
+```typescript
+interface Options {
+    serverUrl?: string;
```

Comment:
dopsat default server url?

Reply:
English: add default server ulr?

Reply:
It's mentioned in Usage tips section at the end, but you're right, I mentioned it also here.